index.html

```diff
@@ -131,6 +133,72 @@ <h1 id="placeholder-img-head">Profesional Art Printing Data</h1>
 </button>
 </div>
 </div>
+</section>
+<section id="desktop-card">
+    <div id="desktop-card-1">
+        <h1>
+        </h1>
+        <p></p>
+        
+    <button class="project-btn-desktop">See Project
+    </button>
+    
+    </div>
+    
+    <div id="desktop-card-2">
+        <h1 id="heading-card-2">Data Dashboard Healthcare
+        </h1>
+        <p>A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard 
+        </p>
+        <ul class="lang-skills">
+            <li class="html">html</li>
+            <li class="bootstrap">bootstrap</li>
+            <li class="Ruby">Ruby</li>
+
+
```

Comment:
- [ ] Please remove these **empty spaces in order to make your codebase  clean**

Reply:
alright done